lib/connection/result/row_stream.js

```diff
@@ -170,6 +175,10 @@ function RowStream(statement, context, options)
         buildMapColumnExtractFnNames(columns, fetchAsString);
     }
 
+    const columnNames = columns
+      .map(column => column.getName())
+    duplicatedColumnNames = columnNames.filter((item, index) => columnNames.indexOf(item) !== index);
```

Comment:
Shouldn't we add some distinction/uniqueness here?

Reply:
This code was removed during refactor.

Reply:
Seems to still exist?